tests/client/test_base.py

```diff
@@ -480,7 +481,8 @@ def test_call_raises_canvas_api_error_with_attributes_on_non_retry_status(self):
         error_json = {'This is some error in json format!'}
         error_status_code = 404
         canvas_error = self.make_retry_call_with_error_code(error_status_code, max_retries=1, error_json=error_json)
-        
+
+        self.assertIs(type(canvas_error), CanvasAPIError)
```

Comment:
Right, and I was going to do so, but I liked the idea of an exact type check that is referenced in the docs for `assertIsInstance`  (https://docs.python.org/2/library/unittest.html#unittest.TestCase.assertIsInstance).